src/sensor.cpp

```diff
@@ -655,8 +653,9 @@ void log_callback_end( uint32_t fps,
 
     stream_profiles uvc_sensor::init_stream_profiles()
     {
-        // D457 development - std::unordered_set<std::shared_ptr<video_stream_profile>> profiles;
-        std::unordered_set<std::shared_ptr<stream_profile_base>> profiles;
+        std::unordered_set<std::shared_ptr<video_stream_profile>> video_profiles;
+        // D457 development
```

Comment:
What does this comment means here?

Reply:
It is to explain why motion frames (from IMU) are received via UVC, while they have always been received via HID. I have made the comment more explicit.